diff --git a/extensions/rocmfpx-vulkan/backend/vulkan-shaders/mul_mmq.comp b/extensions/rocmfpx-vulkan/backend/vulkan-shaders/mul_mmq.comp
index 24daf6d..e703dd2 100644
--- a/extensions/rocmfpx-vulkan/backend/vulkan-shaders/mul_mmq.comp
+++ b/extensions/rocmfpx-vulkan/backend/vulkan-shaders/mul_mmq.comp
@@ -87,6 +87,6 @@
 #define BK_STEP 1
 #else
 #ifndef BK_STEP
-#define BK_STEP 4
+#define BK_STEP 8
 #endif
 #endif
